### DISCUSSION TITLE
Username and password for tools

### DISCUSSION THREAD

**Discussion by MrJasonBryant** [rdz-oss/BattyBirdNET-Pi#50] [Q&A]
Where can i get the Username and password for tools?  The username and password i setup at the beginning isn't work.  How would i reset it?

**Comment by MrJasonBryant** [comment]
nm. :)

**Comment by rdz-oss** [comment]
The default username and password directly after install for the UI are
birdnet
no password
It is recommended to change that immediatley in the Tools-Settings before opening the system to the internet.

**Comment by alfredodaniel** [comment]
I have the same problem. I log in UI "tools" with username birdnet (no password), it works fine. But when I try to enter in "settings" it doesn't work (at least with birdnet username and no password): --You cannot edit the settings for this installation--. Is there a way to change username and pass from command line?

**Reply by jogib00n** [reply]
Hey, same problem here. Clicking on "Tools" is asking for username and password. When entering birdnet and no password I see the buttons with "Settings" "System Info" and so on. When clicking on Settings it is asking again for username and password but birdnet and no password is not working.

**Comment by jogib00n** [comment]
I solved the problem. I added my password to the birdnet.conf parameter CADDY_PWD=password and restarted the services. After that I was able to go to Settings.

**Comment by rdz-oss** [comment]
Thank you for the info. It appears there was an issue with the empty password that appeared sometimes. Default user and password after installation are now set to 'birdnet' both.
